install_centos.md

```diff
@@ -0,0 +1,123 @@
+# Installing st2 on centos/rhel linux
+
+Follow this guide and we'll lead you through the basic installation and setup steps.
+
+## Setting up repos and fetching required software
+
+First let's update the system and fetch software which will be required during installation and setup process:
+```shell
+sudo yum makecache fast
+sudo yum install -y wget httpd-tools
+```
+
+Second, add repositories and install st2.
+```shell
+## NB! This step provides instructions for centos7 distro
+# Don't forget to use correct distro (el6 - for rhel6/centos6, and el7 - for rhel7/centos7)!
```

Comment:
This should be a separate small doc on how to add staging unstable/stable...
This docs for stable for an and user! Of course they don't work at the moment since bintray is empty :)